### PR TITLE
Update the trigger for rescheduling the monthly meeting

### PR DESCRIPTION
**Pull request by jmbriano**
Hi team, last week I missed the monthly meeting but I was only away for a few days. It would have been nice to trigger the re-scheduling process. Nevertheless, as is, the process only considers this option if more than half of the team can not make it. 

For tis reason, I'm proposing a less strict triggering. Can you please provide your view?

### CODE REVIEW

**Comment by saljuama** [comment]
I'm ok with this, to not set a number. 

Also, If we know earlier than 1 week in advance, why to wait to 1 week prior the meeting? Doing it earlier allows for moving it earlier as an option and not just postponing only
PD: i didn't remember the 1 week in advanced and i triggered the rescheduling already for next one just a while ago :sweat_smile:

File: processes/org-monthly-meetings/org-monthly-meetings.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ These meetings are part of the responsibilities of being an organizers and shoul
...
+* **Check availability**: One week before the meeting, the facilitator should check availability for the upcoming meeting. If any of the organisers of the team can not make it on the scheduled day (neither physically nor remotely), the possibility of having the meeting on a different day could be assessed. An alternative day can be chosen if the attendance would result higher **as long as the participants that could make it in the originally scheduled day can still make it in the new proposed date**. Should there not be any date in the week where the attendance would result higher, then the meeting should take place with the available participants. The facilitator is in charge of coordinating any rescheduling.
```
Comment (daverick) [inline processes/org-monthly-meetings/org-monthly-meetings.md]:
```suggestion
* **Check availability**: One week before the meeting, the facilitator should check availability for the upcoming meeting. If any of the organisers can not make it on the scheduled day (neither physically nor remotely), the possibility of having the meeting on a different day could be assessed. An alternative day can be chosen if the attendance would result higher **as long as the participants that could make it in the originally scheduled day can still make it in the new proposed date**. Should there not be any date in the week where the attendance would result higher, then the meeting should take place with the available participants. The facilitator is in charge of coordinating any rescheduling.
```
redondant, the team is only made of organizer right no.

**Comment by daverick** [comment]
> Also, If we know earlier than 1 week in advance, why to wait to 1 week prior the meeting? Doing it earlier allows for moving it earlier as an option and not just postponing only

As the calendar of the monthly meetings is published well in advance, the organizers will know in advance when they surely cannot make it. The organizer that cannot make it could contact the facilitator of the meeting to let him/her/their know that he won't be able to make it and a reschedule would be welcome. and we are not limited to 1 week in advance ... but we shouldn't go for one month in advance either ;-)

**Comment by saljuama** [comment]
> but we shouldn't go for one month in advance either ;-)

Any guiding principle in mind? For me it would be:
 - common sense
 - avoid inefficiencies and waste in the process

**Pull request merged**